docs/src/pages/overview.md

```diff
@@ -11,28 +11,28 @@ Monika operates by reading everything from the config file. From the configurati
+Anyone who created a website or a backend service, would want them to be fast, robust and perform well at all time. Good design, engineering excellence and proper processes will contribute to these goals. Yet, what is often overlook is the importance of a monitoring tool has on a project successes. Hence the budget for a monitoring tool is often marked as optional rather than must have. This is our motivation. Proper monitoring tools must be setup from the very beginning. Any development team should integrate monitoring tools in their development process. Then they should extend it into production deployment.
 
-With this in mind Monika was created, a sythetic monitoring tool.
+With this in mind, we created Monika, a synthetic monitoring tool.
 
 ## Not just another tool
 
-There are plenty of free monitoring tools online, but many fall short our requirements. Free uptime monitors exists, but these monitors only ping for service availability. Users don't typically use services only with pings. There are also plenty of real time monitoring tools. These tools by their very nature require real users which make them less suitable during development. With Monika, you can instead synthesize load scenarios during development, then use the same scenarios after deployment. Synthetic monitors allow QA teams for instance, to generate complex user flows for qualification. The same flows can be used to ensure production performance, and all without the need to install agents or third party libraries.
+There are plenty of free monitoring tools online, but many fall short of our requirements. Free uptime monitors exists, but they only ping for service availability. Most users don't use services only with pings. There are also plenty of real time monitoring tools. These tools need real users, which make them less suitable during development. Monika can synthesize usage scenarios during development, then use same scenarios in production. Synthetic monitoring tool enable you to generate complex usage flows for quality assurance. Those same flows later on can be deploy to check the production environment. All without the need to install agents or third party libraries.
 
-In addition, Monika will be easy to deploy. You can deploy Monika to your own servers in different regions and receive notifications when the quality of service from those regions degrade. All these features are available for free.
+Besides, Monika is easy to deploy. You can deploy Monika to many servers in different locations. Receives notifications if any of those location experiencing service degradation. All these features are available for free.
```

Comment:
You can deploy Monika to many servers in different locations and receive notifications when your website or server experiences service degradation from those locations.